vmdb/app/helpers/application_helper.rb

```diff
@@ -2680,14 +2680,15 @@ def placeholder_if_present(password)
 
   def render_listnav_filename
     if @lastaction == "show_list" && !session[:menu_click] &&
-       %w(ems_cloud ems_cluster ems_infra host miq_template offline orchestration_stack repository
+       %w(container_node container_service ems_container container_group ems_cloud ems_cluster
```

Comment:
@erinboyd, I think @jrafanie is just groaning over the general antipattern already in place here.  Not something you can or should change in this PR.